src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
```

Comment:
The `margin-right` property is used here and in the `&:last-child` pseudo-class. You could create a mixin for the star element that includes all its styles except the `margin-right`, and then include this mixin inside the `.stars__star` and `.stars__star:last-child` selectors with the appropriate `margin-right` value. This would adhere to the DRY principle.